schema.sql

```diff
@@ -363,7 +379,7 @@ CREATE TABLE IF NOT EXISTS `towns` (
   UNIQUE KEY `name` (`name`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
 
-INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '33'), ('players_record', '0');
+INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '35'), ('players_record', '0');
```

Comment:
34?

Reply:
It seems like on last migration someone forgot to update the `db_version` of the `schema.sql` file. 
We are at `34.lua` which results in setting `db_version` to `35`

Reply:
db_version setting in other migrations is always the penultimate

Reply:
after running the `34.lua` the database updates to `35`
![db_number](https://user-images.githubusercontent.com/3682541/185507078-e5b0f9fa-492a-4b63-b569-77af2f17570c.png)